### PR TITLE
[#IOCOM-708] precondition flag management

### PR DESCRIPTION
**Pull request by Garma00**
#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by michaeldisaro — DISMISSED** [review]

File: __integrations__/__mocks__/fixtures.ts [hunk 1]
```diff
@@ -196,18 +219,61 @@ export const deleteAllCollections = (
...
+
 /**
  * Create DB and collections
  */
 export const createCosmosDbAndCollections = (
-  client: CosmosClient,
-  cosmosDbName: string
-): TE.TaskEither<CosmosErrors, Database> =>
+  cosmosParameters: ClientAndDbName,
+  maybeRemoeContentCosmosParameters: O.Option<ClientAndDbName>
```
Comment (michaeldisaro) [inline __integrations__/__mocks__/fixtures.ts]: typo

File: __integrations__/__mocks__/fixtures.ts [hunk 2]
```diff
@@ -339,6 +409,32 @@ export const fillServices = async (
...
+    TE.of,
+    TE.map(c => new RemoteContentCollection.RemoteContentConfigurationModel(c)),
+    TE.chain(model =>
+      pipe(
+        rcConfigurations,
+        RA.map(m => model.create(m)),
+        RA.sequence(TE.ApplicativePar)
+      )
+    ),
+    TE.map(_ => log(`${_.length} Remote content created`)),
```
Comment (michaeldisaro) [inline __integrations__/__mocks__/fixtures.ts]: I'd give a name to _

File: __integrations__/__mocks__/fixtures.ts [hunk 3]
```diff
@@ -339,6 +409,32 @@ export const fillServices = async (
...
+    TE.mapLeft(_ => {
```
Comment (michaeldisaro) [inline __integrations__/__mocks__/fixtures.ts]: ditto

**Review by michaeldisaro — CHANGES_REQUESTED** [review]

File: __integrations__/__mocks__/fixtures.ts (hunk 2 above)
Comment (Garma00) [reply __integrations__/__mocks__/fixtures.ts]: [f8293a6](https://github.com/pagopa/io-functions-app-messages/pull/45/commits/f8293a6ffb27fe52da557c7fa0c48df2487f73de)

File: __integrations__/__mocks__/fixtures.ts (hunk 3 above)
Comment (Garma00) [reply __integrations__/__mocks__/fixtures.ts]: [f8293a6](https://github.com/pagopa/io-functions-app-messages/pull/45/commits/f8293a6ffb27fe52da557c7fa0c48df2487f73de)

**Review by michaeldisaro — APPROVED** [review]

**Pull request merged**